app/models/topic_tracking_state.rb

```diff
@@ -413,7 +414,6 @@ def self.report_raw_sql(
            c.id as category_id,
            #{category_topic_id_column_select}
            tu.notification_level,
-           us.first_unread_at,
```

Comment:
We're selecting this column but it isn't being used anywhere so we're removing it.